cmd/run.go

```diff
@@ -356,33 +357,40 @@ func getCmdRun(gs *state.GlobalState) *cobra.Command {
 		gs: gs,
 	}
 
+	var exampleText bytes.Buffer
+	exampleTemplate := template.Must(template.New("").Parse(`
+  # Run a single VU, once.
+  {{.}} run script.js
+  
+  # Run a single VU, 10 times.
+  {{.}} run -i 10 script.js
+  
+  # Run 5 VUs, splitting 10 iterations between them.
+  {{.}} run -u 5 -i 10 script.js
+  
+  # Run 5 VUs for 10s.
+  {{.}} run -u 5 -d 10s script.js
+  
+  # Ramp VUs from 0 to 100 over 10s, stay there for 60s, then 10s down to 0.
+  {{.}} run -u 0 -s 10s:100 -s 60s -s 10s:0
+  
+  # Send metrics to an influxdb server
+  {{.}} run -o influxdb=http://1.2.3.4:8086/k6`))
+
+	if err := exampleTemplate.Execute(&exampleText, gs.BinaryName); err != nil {
+		gs.Logger.WithError(err).Error("Could not parse binary name")
```

Comment:
It shouldn't fail, you are right. 

Actually this error is related to the `template.Execute()` function, since, as you said, the `os.Executable()` error is already handled. (Did that because I was on autopilot, my bad.)

What is the best option here? My main problem is that I don't when the `template.Execute()` is throwing an error :thinking: 

I can log the error like it's done here:
https://github.com/grafana/k6/blob/bbfe357a39c406e5d47e65fae454fda3847afc50/js/modules/k6/html/gen/gen_elements.go#L323-L341

Or I can just execute the template and "ignore" the error

Let me know what you think!

Reply:
From what I can see in the `template/exec.go` implementation, `Execute()` will return an error if the template is incomplete or empty, or if it fails to write to the passed `io.Writer`. I don't think either can happen in this case, so it's probably fine if we just ignore the error.

@mstoykov WDYT?

Reply:
I feel like if the impossible happens it should not be swept under the rug, so if nothing else this seems like a thing we should log.

I would argue that if this suddenly becomes less impossible it will nigh impossible (:wink:) to debug it if we don't at least log it.

I don't know what the message should be though.

Maybe change that to something like "error during help example generation" :shrug: .

Reply:
Yeah, better safe than sorry I guess!

I just updated the PR. I changed the error for something generic across all the commands.
If you feel like a specific error should be logged in each case, or if you find any other thing that could be done better, just let me know! :smiley:

Reply:
Fair enough. The latest change LGTM. :+1: